app/src/staging/kotlin/uk/gov/onelogin/developer/DeveloperRoutes.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.onelogin.developer
+
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.composable
+import androidx.navigation.navigation
+import uk.gov.onelogin.ui.error.ErrorRoutes.navigateSingleTopTo
+
+object DeveloperRoutes {
+    const val ROOT: String = "/developer"
+    const val START: String = "$ROOT/start"
+
+    fun NavGraphBuilder.developerFlowRoutes(navController: NavHostController) {
+        if (DeveloperTools.isDeveloperPanelEnabled()) {
```

Comment:
The downside is that you have this file duplicated. You have not in `build` and one in `staging`

Reply:
maybe leave this file in the `src/main`?